conbench/job.py

```diff
@@ -155,7 +155,7 @@ class ResultTimeseries:
 SHUTDOWN = False
 _STARTED = False
 
-BMRT_CACHE_SIZE = 0.5 * 10**6
+BMRT_CACHE_SIZE = 0.8 * 10**6
```

Comment:
see commit msg

Reply:
Should we make this a configurable setting by server?

Reply:
> Should we make this a configurable setting by server?

Considering today we do not have the need for a per-deployment setting.

It may become be a candidate for user-facing config at some point in the future. 

There are many ways/ideas though to make "do the right thing" / dynamically adjust to any environment, so changes are this never needs to be user-tunable.

Let's see :)